map_pua_emoji.py

```diff
@@ -53,8 +53,8 @@ def add_pua_cmap(source_file, target_file):
     """Add PUA characters to the cmap of the first font and save as second."""
     font = ttLib.TTFont(source_file)
     cmap = font_data.get_cmap(font)
-    for pua, (ch1, ch2) in (add_emoji_gsub.EMOJI_KEYCAPS.items()
-                            + add_emoji_gsub.EMOJI_FLAGS.items()):
+    for pua, (ch1, ch2) in (list(add_emoji_gsub.EMOJI_KEYCAPS.items())
+                            + list(add_emoji_gsub.EMOJI_FLAGS.items())):
```

Comment:
you could also use `itertools.chain(add_emoji_gsub.EMOJI_KEYCAPS.items(), add_emoji_gsub.EMOJI_FLAGS.items())`